osu.Desktop/DiscordRichPresence.cs

```diff
@@ -167,7 +167,9 @@ private void updatePresence(bool hideIdentifiableInformation)
                 presence.State = clampLength(activity.Value.GetStatus(hideIdentifiableInformation));
                 presence.Details = clampLength(activity.Value.GetDetails(hideIdentifiableInformation) ?? string.Empty);
 
-                if (getBeatmapID(activity.Value) is int beatmapId && beatmapId > 0)
+                if (getBeatmapID(activity.Value) is int beatmapId
+                    && beatmapId > 0
+                    && !(activity.Value is UserActivity.EditingBeatmap && hideIdentifiableInformation))
```

Comment:
This is back-to-front coding, `UserActivity` should expose a method for this rather than hardcoding the types in here. E.g. `ShowsBeatmapInfo(bool hideIdentifiableInformation = false)`

Reply:
probably call it `CanShowBeatmapInfo` or `ShouldShowBeatmapInfo`, but agree with the change.